jenkins_pipelines/scripts/kafka/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM apache/kafka:3.7.1
+
+USER root
+
+RUN \
+  apk \
+    update \
+  && \
+  apk \
+    add \
+      git=2.43.5-r0 \
+      python3=3.11.10-r0 \
+      python3-dev=3.11.10-r0 \
+      py3-pip=23.3.1-r0 \
+      gcc=13.2.1_git20231014-r0 \
+      g++=13.2.1_git20231014-r0 \
+      librdkafka-dev=2.3.0-r1
+
+USER appuser
+
+COPY --chown=appuser "consumer.py" "producer.py" "/home/appuser/"
+
+WORKDIR "/home/appuser"
+
+RUN \
+  python3 \
+    -m \
+      venv \
+      "/home/appuser/venv" \
+  && \
+  . \
+    "/home/appuser/venv/bin/activate" \
+  && \
+  pip \
```

Comment:
wondering if we could use a SUSE package rather than using pip

Reply:
There are packages like this [one](https://packagehub.suse.com/packages/python-confluent-kafka/) available, but it would require registering SUSE product in the container and then enabling the SUSE Package Hub extension ([link](https://packagehub.suse.com/how-to-use/)). I would rather do not use SUSE credentials to make it run.

Reply:
ok